### DISCUSSION TITLE
Are the current mu-plugins used all in production?

### DISCUSSION THREAD

**Discussion by uhlhosting** [szepeviktor/wordpress-website-lifecycle#7] [Q&A]
Hi,

Amazing collection you got here, was wondering if the wordpress side of it, is. up to date.

**Comment by szepeviktor** [comment]
Hello @uhlhosting! 👋🏻 

There are old ones.
WordPress + Viktor = **controversial** - Sometimes I'm forced to run WordPress.
This repository consists of documents. The `mu-plugins/` is my source of MU plugins.
Please read the source code of all of them, tailor them for your needs and use them as necessary.

**Reply by szepeviktor** [reply]
The _read the source code_ part is the most important, the knowledge.
These plugins are dangerous if you don't know them. 🔪🩸

**Reply by szepeviktor** [reply]
I have difficulties expressing myself.
All my stuff assume there is a viktor-like person non-stop caring about the website.

I would gladly ❤️ show you how I build all the bits and pieces for a website. (without fees)
https://github.com/szepeviktor/debian-server-tools/blob/master/Onboarding.md
